rails/app/policies/development_policy.rb

```diff
@@ -12,17 +12,17 @@ def create?
   end
 
   def update?
-    if !user&.disabled? && (user&.admin? || (user&.verified? && user.developments.include?(record)) || (user&.municipal? && user.developments.include?(record)))
-      true
-    else
-      [:flag]
-    end
+    !user&.disabled? && (user&.admin? || (user&.verified? && user.developments.include?(record)) || (user&.municipal? && user.developments.include?(record)))
   end
 
   def destroy?
     !user&.disabled? && (user&.admin? || (user&.municipal? && (record.municipal == user.municipality)))
   end
 
+  def flag?
```

Comment:
Flag is a boolean that any registered user can submit about a project to "flag" it as something being wrong with it. Since generally projects cannot be updated by users that did not create them (or the other criteria enumerated above) I needed to create a conditional to allow anyone to be able to "flag" a development by referencing this policy in the controller and then creating the method in the policy. I tried to find a better way of doing this inside the update policy but failed to find a good example/way that worked. This should remedy the permissions issue going forward in conjunction with the controller conditional.